database/postgres/save_data.py

```diff
@@ -0,0 +1,36 @@
+from database.session import SessionLocal
+from database.models import Sport
+from typing import Dict
+from database.postgres.postgres_team import save_teams
+from database.postgres.postgres_league import save_leagues
+from database.postgres.postgres_game import save_games
+
+def save_api_data(json_data: Dict, sport_name: str ) -> None:
+    session = SessionLocal()
+    try:
+        sport = session.query(Sport).filter_by(sport_name=sport_name).first()
+        if sport:
+            sport_id = sport.sport_id
+        else:
+            print(f"added new sport_id for {sport_name}.")
+            sport_entry = Sport(sport_name)
+            session.add(sport_entry)
+            session.commit()
+            sport_id = sport_entry.sport_id
+
+        entity = json_data.get("get")
+        if entity == "teams":
```

Comment:
We will have it replaced with models and think of a better way of doing this, but:

In general, when you expect some string to be present somewhere (doesn't matter where from exactly) you MUST specify that string somewhere depending on it's scope.

For example:

In your case you have your JSON data passed from API layer directly, which means that this string (let's say "teams") is somewhat global and might be used for comparison somewhere else in the code. If it's global (or expected to be used in diferent modules) we MUST create a separate class/module/whatever which will hold a set of predifined string to use.

For example:

`
class Strings:
    TEAMS = 'teams'
    LEAGUES = 'leagues'
    ...
`